models/user.server.model.js

```diff
@@ -85,8 +87,8 @@ deleteUserById = function(id, done) {
  * @param done function callback, returns status code, and updated document data or message if error
  */
 updateUserById = function(id, updates, done) {
-    if ("hashedPassword" in updates) {
-        updates.hashedPassword = hashPassword(updates.hashedPassword);
+    if ("password" in updates) {
+        updates.hashedPassword = hashPassword(updates.password);
```

Comment:
```suggestion
    if ("plaintextPassword" in updates) {
        updates.hashedPassword = hashPassword(updates.plaintextPassword);
```
If we are going to use a more readable name then why not use plainText to more clearly define that it is not hashed yet

Reply:
Sounds good to me, done.